main.go

```diff
@@ -50,6 +50,7 @@ var (
 	connectCert     = connect.Flag("cert", "Client certificate chain for connecting to server (PEM).").ExistingFile()
 	connectKey      = connect.Flag("key", "Private key for client certificate, if not in same file (PEM).").ExistingFile()
 	connectStartTLS = connect.Flag("start-tls", "Enable StartTLS protocol ('ldap', 'mysql', 'postgres', 'smtp' or 'ftp').").Short('t').PlaceHolder("PROTOCOL").Enum("mysql", "postgres", "psql", "smtp", "ldap", "ftp")
+	connectAs       = connect.Flag("asuser", "With starttls, connect as this db user (or smtp ehlo)").Default("certigo").String()
```

Comment:
Flag/description here is a bit confusing to me. I'd prefer something like `--identity` and `With --starttls, sets the DB user or SMTP EHLO name`. Also add validation to make sure this isn't used w/o `--starttls`.